app/views/content/blog/my-first-week-as-a-trainee-teacher.md

```diff
@@ -1,48 +1,68 @@
+---
+title: My first week as a trainee teacher
+author: Cindy Ventura
+date: "2021-09-20"
+images:
+  cindy-ventura:
+    path: "media/images/content/blog/cindy-ventura.jpg"
+    alt: "A photograph of trainee teacher Cindy Ventura"
```

Comment:
@gemmadallmandfe - any hints on improving this alt text?

Reply:
Alt text best practice is not to start with 'A photograph of'

How about:

Trainee teacher Cindy Ventura wearing a red polo shirt

(or you could just put 'Trainee teacher Cindy Ventura')